client/src/AppBundle/Resources/views/Report/Document/step2.html.twig

```diff
@@ -24,9 +24,15 @@
 
     <h2 class="heading-medium">{{ (page ~ '.selectHeading') | trans }}</h2>
 
-    <div class="text">
-        <p>{{ (page ~ '.selectHelp') | trans }}</p>
-        <p>{{ (page ~ '.selectHint') | trans }}</p>
+    <p>{{ (page ~ '.selectHelp') | trans }}</p>
+
+    <div class="govuk-inset-text">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                <b>{{ (page ~ '.selectHint1') | trans }}</b>
```

Comment:
Ol' fusspot strikes again
```suggestion
                <p><strong>{{ (page ~ '.selectHint1') | trans }}</strong></p>
```

Reply:
You can see my extent of touching anything frontend has been VERY LIMITED of late 😂